src/docs/Components/Accordion/code.mdx

```diff
@@ -0,0 +1,98 @@
+---
+title: 'Code'
+order: 2
+---
+
+## Import
+
+Import *the settings* and *the accordion* scss files. 
+```scss
+@import 'settings-tools/_all-settings';
+
+@import 'components/_c.accordion';
+```
+
+## Basic usage
+To create a basic accordion, wrap a `checkbox`, a `label` and a `div` inside a `div` and apply the following classes: 
+
+- `mc-accordion` on a div container
+- `mc-accordion__trigger` on a input[checkbox] tag
+- `mc-accordion__label` on a label tag
+- `mc-accordion__content` on a div tag
+
+```html
+<div class="mc-accordion">
+  <input type="checkbox" id="basicUsage" class="mc-accordion__trigger" />
+  <label for="basicUsage" class="mc-accordion__label" aria-expand="false">
+    Accordion title
+    <span class="mc-accordion__icon"></span>
+  </label>
+  <div class="mc-accordion__content">
+    <p>
+      Fusce iaculis dolor nulla, a maximus ipsum sollicitudin et. Ut condimentum
+      at orci aliquam feugiat. Curabitur sagittis placerat leo sit amet
+      pharetra.
+    </p>
+  </div>
+</div>
+```
+
+<Preview path="basic" />
+
+## States
+
+Depending on the context of use, the accordion can have several states: 
+
+- Close
+- Open 
+- Close Disabled
+- Open Disabled
+
+<Preview path="open" />
+<Preview path="disable" />
```

Comment:
I think you can give examples of each of the 4 states in a single preview file.
Example => http://mozaic.adeo.cloud/previews/Foundations/Typography/HeadingStyles/previews/typographyHeading-sizes.html